src/logic/wallets/getWeb3.ts

```diff
@@ -11,6 +12,8 @@ import { getRpcServiceUrl, _getChainId } from 'src/config'
 import { CHAIN_ID, ChainId } from 'src/config/chain.d'
 import { isValidCryptoDomainName } from 'src/logic/wallets/ethAddresses'
 import { getAddressFromUnstoppableDomain } from './utils/unstoppableDomains'
+import { sameString } from '../../utils/strings'
```

Comment:
No need to change it, but we should stop using this function. It's the same as `===`.

Reply:
It converts them both to the same case beforehand.